client/src/store/modules/i18n/uk.js

```diff
@@ -57,16 +57,16 @@ export default {
         'Ви збираєтесь видалити свій обліковий запис Schedulia. Це набере чинності негайно, і ви більше не матимете доступу до даних свого облікового запису.',
     DESCRIPTION_LABEL: 'Опис / інструкції',
     DIALOG_EVENT_TYPE_DELETE_CONFIRM:
-        'Користувачі не зможуть запланувати подальші зустрічі зі видаленими типами подій. Попередньо заплановані зустрічі будуть впливати.',
+        'Користувачі не зможуть запланувати подальші зустрічі зі видаленими типами подій. Попередньо заплановані зустрічі буде затронуто.',
```

Comment:
there should be "з" instead of "зі". Sentence "Попередньо заплановані зустрічі буде затронуто." is incorrect. Use "Це вплине на попередньо заплановані зустрічі."